tests/issue-5466-alert-then-pass-01/test.rules

```diff
@@ -0,0 +1,2 @@
+pass tcp any any -> any 22 (alert; sid:2; gid:10000003; msg:"PASS SSH";)
+drop tcp any any -> any any (noalert; sid:1; rev:1; msg:"DROP all TCP";)
```

Comment:
The sid vs signature order here is sneaky xD

Reply:
hehe yeah, just copied and updated the 4663-03 test, so came from there :)